Sources/Views/BondiBallView.swift

```diff
@@ -24,32 +24,24 @@
 
 import UIKit
 
-internal final class PaintBallView: UIView {
+internal final class BondiBallView: UIView {
   override public init(frame: CGRect) {
     super.init(frame: frame)
-    self.setup()
+    self.isOpaque = false
+    self.layer.shadowColor = UIColor.black.cgColor
+    self.layer.shadowOpacity = 1
+    self.layer.shadowRadius = 5
+    self.layer.shadowOffset = .zero
   }
 
   @available(*, unavailable)
   public required init?(coder: NSCoder) {
     fatalError()
   }
 
-  func setup(){
-    self.isOpaque = false
-    self.layer.shadowColor = UIColor.label.cgColor
-    self.layer.shadowOpacity = 0.25
-    self.layer.shadowRadius = 3
-    self.layer.shadowOffset = CGSize(width: 3.0, height: 3.0)
-  }
-
-  override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
-    setup()
-  }
-
   override public func draw(_ rect: CGRect) {
```

Comment:
Research how this draw function works and fix the weird layout when there is a very small grid

Reply:
https://github.com/DreamEmulator/Fidget/issues/6